packages/cli/src/cmds/assert/strategy/methodStrategy.ts

```diff
@@ -0,0 +1,12 @@
+// @ts-ignore
+import { Event } from '@appland/models';
+import Strategy from './strategy';
+import { Scope } from '../types';
+
+export default class MethodStrategy extends Strategy {
+  protected scope: Scope = 'method';
+
+  protected isEventApplicable(event: Event): Boolean {
+    return event.methodId;
```

Comment:
If you want to select just the events which are functions, try `Event.isFunction`